### ISSUE TITLE
FR: option to show not all forecast items, but only every 2nd/3rd/...

### ISSUE THREAD

**Issue by Masterz69** [mlamberts78/weather-chart-card#166]
**Is your feature request related to a problem? Please describe.**
In my specific case would like to see longer forecast, but sceern width is not enough when every hour forecast shown.

**Describe the solution you'd like**
Option to skip forecast items and show every 2nd/3rd/4th/...

**Additional context**
Thank You very much - card looks great.

**Comment by dxmnkd316** [comment]
Agreed with Masterz69 and second the request.  This card is pretty great.  It's very nearly what I would consider a spiritual successor to some of the older Weather Underground apps before they were turned to revenue-generators.

If you need anyone to help test, please let me know.

**Comment by dxmnkd316** [comment]
Would still like to see this integrated.

**Comment by dxmnkd316** [comment]
Would still like to see this integrated.

**Comment by BlackTowerWA** [comment]
I would like this same feature. Pirate Weather gives every hour in the hourly forecast so I can't see as far ahead as I'd like in the hourly graph. Allowing the option to only show ever nth hour would let this simulate weather providers that only give every 3 or 6 hours.
